src/step-input.scss

```diff
@@ -84,6 +84,15 @@ $block: #{$fd-namespace}-step-input;
     }
   }
 
+  &__container {
+    display: flex;
+    align-items: center;
+
+    .fd-form-label {
```

Comment:
You can't use this directly here as it's violating the self-contained rules.

Reply:
removed.